tests/bsim/bluetooth/audio/test_scripts/has.sh

```diff
@@ -24,3 +25,18 @@ Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
   -D=2 -sim_length=60e6 $@
 
 wait_for_background_jobs
+
+printf "\n\n Running Preset Changed Offline Behavior test \n\n"
+
+SIMULATION_ID="has_offline_behavior"
+
+Execute ./bs_${BOARD}_tests_bsim_bluetooth_audio_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -testid=has_offline_behavior -rs=24
+
+Execute ./bs_${BOARD}_tests_bsim_bluetooth_audio_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=1 -testid=has_client_gatt -rs=46
+
+Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
+  -D=2 -sim_length=60e6 $@
+
+wait_for_background_jobs
```

Comment:
Suggest to put this into a new script, (e.g. `has_offline.sh` so that it can run in parallel the "regular" HAS test